src/api/base.tsx

```diff
@@ -0,0 +1,14 @@
+import { create } from "apisauce";
+
+const api = create({
+  baseURL: "https://",
+  headers: {
+    // eslint-disable-next-line prettier/prettier
```

Comment:
Should also add in `Accept` and `Content-Type` and set both their values to be `application/json`